Sample/Sample.csproj

```diff
@@ -22,6 +22,9 @@
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">14.2</SupportedOSPlatformVersion>
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">24.0</SupportedOSPlatformVersion>
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'">13.1</SupportedOSPlatformVersion>
+
+		<MtouchLink>SdkOnly</MtouchLink>
+		<MtouchExtraArgs>$(MtouchExtraArgs) --weak-framework=NewsstandKit</MtouchExtraArgs>
```

Comment:
what is the purpose of this line?

Reply:
it's a workaround for Xcode 15 and iOS 17 API support as stated in one of the commit messages: https://github.com/dotnet/maui/discussions/15540

Reply:
did you try to run app on ios 17 without this lines?

Reply:
yes, it results in the error for which the workaround is intended in Visual Studio 17.7.6 with the latest MAUI workloads:

```
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.60\targets\Xamarin.Shared.Sdk.targets(1226,3): error : clang++ exited with code 1:
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.60\targets\Xamarin.Shared.Sdk.targets(1226,3): error : ld: warning: ignoring duplicate libraries: '-lSystem.IO.Compression.Native', '-lSystem.Native', '-lSystem.Net.Security.Native', '-lSystem.Security.Cryptography.Native.Apple', '-lcompression', '-lmono-component-debugger', '-lmono-component-diagnostics_tracing', '-lmono-component-hot_reload', '-lmonosgen-2.0', '-lxamarin-dotnet-debug'
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.60\targets\Xamarin.Shared.Sdk.targets(1226,3): error : ld: framework 'NewsstandKit' not found
2>C:\Program Files\dotnet\packs\Microsoft.iOS.Sdk\16.4.60\targets\Xamarin.Shared.Sdk.targets(1226,3): error : clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Note:** The solution cannot be built in Visual Studio 17.8.0 Preview 7.0, because with the release of .NET 8.0, .NET 6.0 will be out of support and thus throws errors, which in Visual Studio 17.7.6 are still warnings. It would make sense to remove the .NET 6.0 targets and switch to at least .NET 7.0. I just didn't do this, because I only wanted to add the plain .net6.0 and .net7.0 targets to be able to support unit test projects.

Reply:
BTW, the workaround only applies to the sample application and doesn't affect the library at all.